conventions/README.md

```diff
@@ -95,12 +95,12 @@ To make this work, inside a component's SASS file, _you must use colors from the
 ```scss
 // 🙅‍♀️ using the sass var will not correctly inherit or change in light/dark mode
 :host {
-  color: $ui-blue-1;
+  color: $ui-brand-light;
```

Comment:
@bstifle, light is used here.

Reply:
I think we should keep these consistent with the CSS variable nomenclature and not use `-light`

CC'ing @paulcpederson

Reply:
Disregard this thread. It's okay as is.

Reply:
Yeah, I think that's just what the sass variable is called now they added light to all the light theme colors